modules/understanding-upgrade-channels.adoc

```diff
@@ -7,159 +7,146 @@
 // * updating/updating-disconnected-cluster.adoc
 
 [id="understanding-upgrade-channels_{context}"]
-= {product-title} Upgrade Channels and Releases
+= {product-title} upgrade channels and releases
 
 In {product-title} 4.1, Red Hat introduced the concept of channels for
 recommending the appropriate release versions for cluster upgrade. By controlling
-the pace of upgrades, these upgrade channels allow users to choose an upgrade
+the pace of upgrades, these upgrade channels allow you to choose an upgrade
 strategy. Upgrade channels are tied to a minor version of
 {product-title}. For instance, {product-title} {product-version}
-upgrade channels will never include an upgrade to a 4.5 release. This ensures
-administrators make an explicit decision to upgrade to the next minor version of
-{product-title}. Upgrade channels only control release selection and have
-no impact on the version of the cluster you install; the `openshift-install`
-binary for a given patch level of {product-title} always installs that patch level.
+upgrade channels will never include an upgrade to a 4.5 release. This strategy ensures that
```

Comment:
Hmm, is master now tracking 4.5, in which case this should be bumped to "a 4.6 release"?

Reply:
No, docs master is still tracking 4.4.